src/Perpetuum/Services/Channels/ChatCommands/GameAdminCommands.cs

```diff
@@ -30,21 +31,48 @@ public AdminCommandRouter(GlobalConfiguration configuration, ISessionManager ses
 
         public void TryParseAdminCommand(Character sender, string text, IRequest request, Channel channel, IChannelManager channelManager)
         {
-            if (IsAdminCommand(sender, text))
+            var b_isAdmin = IsAdmin(sender);
```

Comment:
I know its a nit, but style.
Use the https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions style or what the rest of the project uses (predominantly)
PascalCase for methods, classes, etc.
camelCase for parameters, vars, etc.
_camelCase for privates
Never snake_case

Is it 100% consistent in this repo?  Definitely not.  But do your best to match the style of the files touched.

Reply:
If the command succeeds: log it, admin or otherwise
It should be the handlers job to make sure non-admins can't invoke commands, but should they do so they should also be in the log